<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BilateralFilterDialog</class>
 <widget class="QDialog" name="BilateralFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="lyt_diameter" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_diameter_label">
       <property name="text">
        <string>Diameter of each pixel:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_diameter_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyt_sigma_color" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_sigma_color_label">
       <property name="text">
        <string>Sigma Color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spb_sigma_color_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyt_sigma_space" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_sigma_space_label">
       <property name="text">
        <string>Sigma Space:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spb_sigma_space_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
